atest/robot/test_libraries/logging_api.robot

```diff
@@ -15,9 +15,13 @@ Log levels
     Check log message  ${tc.kws[1].msgs[6]}  info msg 2   INFO
     Check log message  ${tc.kws[1].msgs[7]}  warn msg 1   WARN
     Check log message  ${tc.kws[1].msgs[8]}  warn msg 2   WARN
+    Check log message  ${tc.kws[1].msgs[9]}  error msg 1   ERROR
+    Check log message  ${tc.kws[1].msgs[10]}  error msg 2   ERROR
     Check log message  ${ERRORS.msgs[0]}     warn msg 1   WARN
     Check log message  ${ERRORS.msgs[1]}     warn msg 2   WARN
-
+    Check log message  ${ERRORS.msgs[2]}     error msg 1   ERROR
+    Check log message  ${ERRORS.msgs[3]}     error msg 2   ERROR 
+       
```

Comment:
What IDE do you use? I have a mac and have had a hellacious time getting ride to work because wxpython packaging is a nightmare.